README.md

```diff
@@ -99,4 +98,143 @@ have to create the following secrets:
 SUPABASE_ACCESS_TOKEN: Your personal access token, can be found at https://supabase.com/dashboard/account/tokens
 SUPABASE_DB_PASSWORD: Your supabase database password
 SUPABASE_PROJECT_ID: Your supabase project id
+```
+
+
+## Customizing
+
+You can customize the title, logo, theme, and domain of the CRM app by passing custom props to the `<CRM>` component:
+
+```tsx
+// App.tsx
+import React from 'react';
+import CRM from './CRM';
+
+const App = () => (
+    <CRM 
+        title="Custom CRM Title" 
+        logo="custom-logo.png" 
+    />
+);
+
+export default App;
+```
```

Comment:
The improvements to the docs look good, but did you intend to include them in this PR? Neither the title nor the description speaks about that...